pkg/controller/storagecluster/storagecluster.go

```diff
@@ -242,6 +243,8 @@ func (c *Controller) Reconcile(ctx context.Context, request reconcile.Request) (
 		return reconcile.Result{}, err
 	}
 
+	c.registerCSRAutoApproval(cluster)
```

Comment:
Can we also unregister on StorageCluster delete so we no longer approve any further/pending CSRs with that ID? 
We can do that in the `deleteStorageCluster` function -  https://github.com/libopenstorage/operator/blob/ff3634adc3ffd93ad17dc7ba0e92d55682fc57a1/pkg/controller/storagecluster/storagecluster.go#L649